src/site/content/en/blog/image-support-for-async-clipboard/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Image support for the async clipboard API
+subhead: The Asynchronous Clipboard API now handles some images, in addition to text.
+authors:
+  - thomassteiner
+description: Chrome 76 adds expands the functionality of the Async Clipboard API to add support for png images. Copying and pasting images to the clipboard has never been easier.
+date: 2019-07-03
+updated: 2019-10-08
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - chrome76
+  - cutandcopy
+  - execcommand
+  - input
+  - clipboard
+hero: hero.jpg
+alt: Clipboard with shopping list
+draft: true
+---
+
+Chrome 66 added support for the
+[text portion](https://developers.google.com/web/updates/2018/03/clipboardapi)
+of the [Asynchronous Clipboard API](https://w3c.github.io/clipboard-apis/#async-clipboard-api).
```

Comment:
I think we usually prefer to link to MDN when we mention an API. I realize [the MDN page for Clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) is a little short tho (you have to click on the Clipboard interface to see the methods). @kaycebasques do you have an opinion here?

Reply:
```suggestion
of the [Asynchronous Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API).
```
I think MDN aligns more closely with our reader's goals. I understand that this is a capabilities post, and the readers might be more bleeding-edge and therefore might be interested in a spec, but I think we should optimize for the typical web developer who just heard about a cool API and wants to build stuff with it. For that typical reader MDN is the better destination.

I updated the checklist to mention this: https://github.com/GoogleChrome/web.dev/commit/ae7e7d0a70bf22540d10266c2280d5fd70ba8326